libs/scripts/scripts.js

```diff
@@ -149,7 +150,11 @@ const eagerLoad = (img) => {
 
 (async function loadPage() {
   performance.mark('loadpage');
+  window.addEventListener('load', (e) => {
+    e.preventDefault();
+  });
   setConfig(config);
   loadLana({ clientId: 'milo' });
   await loadArea();
+  scrollToHashedElement();
```

Comment:
The scripts are defined on a per-consumer basis - so every consumer would need to redefine this, I don't think this is the right place to call this function, otherwise each consumer would need to repeat this